### DISCUSSION TITLE
Cake.grate is ready to be released to Nuget ... but how?

### DISCUSSION THREAD

**Discussion by franhoey** [cake-build/cake#4287] [Extension requests]
I've completed the [Cake.grate](https://github.com/cake-contrib/Cake.grate) extension and I had started to build a cake script and github action to deploy it on the [pipeline branch](https://github.com/cake-contrib/Cake.grate/tree/feature/pipeline). However, the Cake.grate [nuget package](https://www.nuget.org/packages/Cake.grate) appears to have been setup when the repo was brought into cake-contrib 

So ... how to I proceed now? I was following at Cake.ESLint, but Cake.Recipe seemed a little obfuscating to me, so I wanted to use plain old Cake, was that the wrong decision?

The project was brought into cake-contrib here - https://github.com/orgs/cake-build/discussions/4152

**Comment by nils-a** [comment]
Definitely not a wrong decision. Regardless of whether you go with Cake.Recipe or something else - everything is fine.

@augustoproiete probably tried to reserve the NuGet package.

@augustoproiete could you transfer Cake.grate to @franhoey (and add the cake-contrib user)?

**Reply by augustoproiete** [reply]
Thanks @franhoey for your contribution and @nils-a for helping out.

Indeed, I reserved the [Cake.grate](https://www.nuget.org/packages/Cake.grate) package for future use.

- @franhoey Send me your NuGet username, so I can add you as owner of the Cake.grate NuGet package
- @nils-a I've added `cake-contrib` to it just now

**Reply by franhoey** [reply]
Thank you @augustoproiete. My username is [franhoey](https://www.nuget.org/profiles/franhoey)

**Reply by augustoproiete** [reply]
@franhoey Sent you the invite to the Cake.grate NuGet package

**Reply by franhoey** [reply]
I've got it, thank you
